deploy/gatekeeper.yaml

```diff
@@ -3,24 +3,26 @@ kind: Namespace
 metadata:
   labels:
     control-plane: controller-manager
-    controller-tools.k8s.io: "1.0"
   name: gatekeeper-system
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.2
```

Comment:
let's add libs here:

https://github.com/open-policy-agent/gatekeeper/blob/4df5adc49cb202d2251d1b94388cd840243eac96/deploy/gatekeeper.yaml#L419-L421

https://github.com/open-policy-agent/frameworks/blob/e6de551bda0b30151ef166c1ac9f988b585600c7/constraint/config/crds/templates_v1beta1_constrainttemplate.yaml#L47-L50

Reply:
That will happen automatically once I am no longer overriding frameworks with my local copy. Currently it appears the modvendor "use the cache" solution doesn't respect gomod